### PR TITLE
Feature/python device client

### PR DESCRIPTION
**Pull request by yowpark**
# PR Template

## Checklist

- [ ] E2E finished.
- [x] Local, Self Hosted and Development environment test finished.
- [x] Local, E2E, Self Hosted, Development and Production env variable checked.
- [ ] (Backend only) DB migration file creation checked.
- [ ] **If this is a API deleted or changed**: Check side effects for API deletion or related code change.

- https://github.com/dogu-team/dogu/issues/362 https://github.com/dogu-team/dogu/issues/344

### CODE REVIEW

**Comment by yowpark** [comment]
sample routine
```
name: gamium-py
on:
  workflow_dispatch:
jobs:
  job-sample:
    runs-on:
      group: 
        - android
        - ios
    steps:
      - name: prepare
        uses: dogu-actions/prepare
        with:
          appVersion:
            android: 2.0.5
            ios: 2.0.5
      - name: run test
        run: cd gamium/python/pytest && python3 -m venv .venv && source .venv/bin/activate && pip install -r requirements.txt && pytest app/test_dogurpgsample.py --capture=no
    record: true
```

Connected github repo: https://github.com/dogu-team/dogu-routine-examples

File: packages/python/dogu-device-client/DEV.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# pytest-dogu-sdk
```
Comment (oneofthezombies) [inline packages/python/dogu-device-client/DEV.md:1]: please rename titme

File: packages/python/dogu-device-client/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# dogu gamium client
```
Comment (oneofthezombies) [inline packages/python/dogu-device-client/README.md:1]: please rename title

**Comment by oneofthezombies** [comment]
LGTM!!

**Pull request merged**